Source/ActionController.swift

```diff
@@ -25,6 +25,13 @@
 import Foundation
 import UIKit
 
+#if swift(>=4.2)
+import UIKit.UIGeometry
+extension UIEdgeInsets {
+    public static let zero = UIEdgeInsets()
```

Comment:
Please remove the `public` here

Reply:
I removed this altogether and directly fixed the problem it was supposed to resolve.